src/BunqLib.php

```diff
@@ -90,8 +89,13 @@ public function __construct(BunqEnumApiEnvironmentType $bunqEnumApiEnvironmentTy
 
     /**
      * Restores the context from the saved file during creation.
+     *
+     * @param bool $resetConfigIfNeeded
+     *
+     * @throws BunqException
+     * @throws ForbiddenException
      */
-    private function setupContext()
+    private function setupContext(bool $resetConfigIfNeeded = true)
```

Comment:
No typehint for bool.

Reply:
This is not the backend tho :P here we follow the psr style guide. AFAIK there is no mention of not type hinting bools 🤔

Reply:
@OGKevin as long as you don't suffer from the weird casting shizzle I'm fine with it.

Reply:
@sandervdo nope we don't suffer from this 😬

Reply:
@OGKevin then fine.